frontend/views/pages/GroupChat.vue

```diff
@@ -1,10 +1,54 @@
 <template lang='pug'>
 page(pageTestName='dashboard' pageTestHeaderName='groupName')
-  template(#title='') {{ L('Group Chat') }}
+  template(#title='')
+    .c-header
+      i(
+        v-if='summary.private !== undefined'
+        :class='`icon-${ summary.private ? "lock" : "hashtag" } c-group-i`'
+      )
+      | {{summary.title}}
+      menu-parent
+        menu-trigger.c-menu-trigger.is-icon-small
+          i.icon-angle-down.c-menu-i
+
+        menu-content
+          menu-header
+            i18n Channel Options
+
+          ul
+            menu-item(@click='openModal("EditChannelNameModal")')
+              i18n Rename
+            menu-item(@click='openModal("GroupMembersAllModal", {name: summary.title})')
+              i18n Members
+            menu-item.c-separator(tag='router-link' itemid='hash-3' to='/group-settings')
+              i18n Notifications settings
+            menu-item(@click='openModal("LeaveChannelModal")')
+              i18n(:args='{ groupName: summary.title }') Leave {groupName}
+            menu-item.has-text-danger(@click='openModal("DeleteChannelModal")')
+              i18n Delete channel
+
+  template(#description='')
+    .c-header-description
+      button.is-unstyled.c-link(
+        @click='openModal("GroupMembersAllModal")'
+      ) {{ members.size + ' ' + L('members') }}
+      | ∙
+      .is-unstyled.c-link(
+        tag='button'
+        v-if='summary.description'
+        @click='openModal("EditChannelDescriptionModal")'
+      )
+        | {{ summary.description }}
+        i.icon-pencil-alt
+      i18n.is-unstyled.c-link(
+        tag='button'
+        v-else
+        @click='openModal("EditChannelDescriptionModal")'
+      )
+        | Add description
```

Comment:
Even if this syntax works, it's really weird and I'm worried it might break our string extraction stuff at some point. Can you put this string on line 47 after the `)`?